kythe/java/com/google/devtools/kythe/analyzers/base/EdgeKind.java

```diff
@@ -38,6 +38,7 @@
   EXPORTS("exports"),
   EXTENDS("extends"),
   GENERATES("generates"),
+  IMPUTES("imputes"),
```

Comment:
This should go above as one of the anchor edges.

Reply:
done